### DISCUSSION TITLE
Host not ready to accept connections - please try again later (NOS 2.8.7)

### DISCUSSION THREAD

**Discussion by kennethfugate** [kej715/DtCyber#155] [Q&A]
I followed all of the installations instructions using the defaults for NOS 2.8.7 on a Raspberry Pi 3 B+.  When I try to run iaf using the port 8001 web interface, the system shows "Host not ready to accept connections - please try again later" and won't go any further.  How do I fix this so I can interact with the system?  Do I need to enable some ports?  Something else? Run a command in the operator console?

Thanks, Ken

**Comment by kej715** [comment]
"Host not ready to accept connections" indicates that NAM has not yet
informed the frontend processor that it has fully initialized and is ready
to begin networking operations. Usually, this state is very brief during
system startup. However, if NAM is unable to start successfully one or both
of its supervisory applications, CS (Communication Supervisor) or NS
(Network Supervisor), this situation will occur too. This can occur if you
attempt to customize the password of the NETOPS account, and something goes
wrong with that. Did you attempt to change the password of NETOPS? If you
enter the following commands at the system console, what is displayed?

K,NAM.

K.ST


Normally, you should see a relatively long list of applications that have
been started, including applications named CS, NS, and NVF. If these are
not displayed, that would explain why the network does not become ready to
accept connections, and it's likely that this is because NAM is not
launching these applications with the correct password for NETOPS. If this
is the case, then it should be possible to correct the problem by executing
the following commands at the system console. This will stop NAM and
restart it in a mode that allows you to inform it of the NETO{PS password
that you want it to use.

IDLE,NAM.

NAMNOGO.

CFO,NAM.UN=NETOPS,PW=password

CFO,NAM.GO

**Comment by kennethfugate** [comment]
It never got to the operator> prompt after 18 hours after doing the sudo node install.  I did another sudo node install rtr nos287-full-875 and after entering the time iaf became available after 10 minutes or so, showing IAF network connected on line 1 of the system status on the operator console.  It is working now, thanks.
